WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -41,6 +46,20 @@ final class SurveyViewController: UIViewController {
         webView.load(request)
         webView.navigationDelegate = self
     }
+
+    /// Adds a loading view to the screen pinned at the center
+    ///
+    private func addLoadingView() {
+        loadingView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(loadingView)
+        view.pinSubviewAtCenter(loadingView)
+    }
+
+    /// Removes the loading view from the screen with a fade out animaition
+    ///
+    private func removeLoadingView() {
+        loadingView.fadeOut()
```

Comment:
nit: not sure if we might want to remove `loadingView` from its superview on fade out completion?